vm/state_reader.go

```diff
@@ -90,3 +90,8 @@ func JunoStateGetCompiledClass(readerHandle C.uintptr_t, classHash unsafe.Pointe
 
 	return unsafe.Pointer(cstring(compiledClass))
 }
+
+func fillBufferWithFelt(val *felt.Felt, buffer unsafe.Pointer) int {
```

Comment:
nit: replace `int` with `C.int` ?

Reply:
Since we are converting the result to C.int everwhere we call it, I guess it looks better. I updated it